src/components/App.js

```diff
@@ -8,6 +8,7 @@ import React, { Component } from "react";
 import { connect } from "react-redux";
 import { features } from "../utils/prefs";
 import actions from "../actions";
+import A11yIntention from "./A11yIntention";
```

Comment:
Decided to introduce a component here, instead of bringing in [what-input](https://github.com/ten1seven/what-input) or a polyfill for `:focus-visible`.

Those solutions are really great, but our usecase here is much simpler than the problems they solve. For example, I don't believe we care about touch or pointer interaction.